### PR TITLE
New: (RarBg) Log Rate Limiting at Info

### PR DESCRIPTION
**Pull request by bakerboy448**
#### Database Migration
NO

#### Description

Most (new) prowlarr users using rarbg are likely not running debug....and this causes extra support across all starrs.   Log no results details at info so it's clear even to users 
#### Todos
- Tests
- Translation Keys (./src/NzbDrone.Core/Localization/Core/en.json)
- [Wiki Updates](https://wiki.servarr.com)

#### Issues Fixed or Closed by this PR

* Fixes #XXXX

### CODE REVIEW

**Comment by Qstick** [comment]
I can see the rate limit being info, but the other seems spammy

File: src/NzbDrone.Core/Indexers/Definitions/Rarbg/RarbgParser.cs [hunk 1]
```diff
@@ -50,7 +50,7 @@ public IList<ReleaseInfo> ParseResponse(IndexerResponse indexerResponse)
 
                     if (jsonResponse.Resource.rate_limit is 1)
                     {
-                        _logger.Debug("No results due to rate limiting. Reason: {0}", reason);
+                        _logger.Info("No results due to rate limiting. Reason: {0}", reason);
```
Comment (bakerboy448) [inline src/NzbDrone.Core/Indexers/Definitions/Rarbg/RarbgParser.cs:53]: Actually should this be a warning instead?

Comment (Qstick) [reply src/NzbDrone.Core/Indexers/Definitions/Rarbg/RarbgParser.cs:53]: As much as they rate limit, no, I don't think so

Comment (bakerboy448) [reply src/NzbDrone.Core/Indexers/Definitions/Rarbg/RarbgParser.cs:53]: should we be throwing ratelimitreached or is https://github.com/Prowlarr/Prowlarr/issues/1277 still occurring?

**Pull request closed without merging**